reana/reana_dev/git.py

```diff
@@ -328,10 +336,13 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
     help="Which components? [shortname|name|.|CLUSTER|ALL]",
 )
 @click.option(
-    "--exclude-components", default="", help="Which components to exclude? [c1,c2,c3]",
+    "--exclude-components",
+    default="",
+    help="Which components to exclude? [c1,c2,c3]",
 )
+@click.option("--target-directory", default=".", help="In which directory install? [.]")
```

Comment:
Note that the default value cannot be `.` since it would break cases of people cloning new repository onto an existing system. What we should do is simply that when `--target-directory` is not present, the code would fall back to the old behaviour of discovering top-level source directory...

Reply:
Agree, the default value could be some falsy value, probably you can even delete the default value to be `None` when `--target-directory` is not passed (I haven't tested it).